z2/src/bin/z2.rs

```diff
@@ -386,6 +389,9 @@ struct JoinStruct {
     /// Specify the ZQ2 chain you want join
     #[clap(long = "chain")]
     chain_name: chain::Chain,
+    /// Specify the container label to run
+    #[clap(long = "container")]
+    container: Option<String>,
```

Comment:
not mandatory change: can be more intuitive to replace "container" with "image" since recall immediately the value of the string?

Reply:
Separate PR else I'll never merge, but good point - thanks .

Reply:
Want to drop me a line on slack? This is called `container` because the parameter it feeds is called `container` and that feeds a Tera variable called `container`, so probably a good plan to work out how far down that stack we'd like the rename to go ..